README.md

```diff
@@ -2,3 +2,13 @@
 
 Holochain is being rewritten in Rust as a set of crates that make up the Holochain Library. This means we need a container app to load Holochain and use its features. Using the cmd line wrapper [CLAP](https://github.com/kbknapp/clap-rs ) it was easy to parse the command line and pass those values to Holochain.
 As the library progresses we will add more exposed functions to the command line.
+
+## Quick How-to
+- have Rust installed
+
+In a terminal
+- clone this repository
+- change directories to this folder
+- run `cargo build`
```

Comment:
If you actually want to install this tool:
```shell
cargo install -f --path .
```

Reply:
Oh, excellent!